woocommerce/woocommerce
# Disable product info in Status Widget when stock management is disabled

There's no need to show the two products cells in Status Widget table when stock management is disabled, otherwise you'll get 2 links pointing nowhere (specific report pages are disabled and you'll get an "unauthorized" error)

### All Submissions:






### Changes proposed in this Pull Request:



Product's stock infos are always displayed in Status Widget, regardless of Stock Management option

When stock management is disabled, the two links contained in product cells of status widget table will point to disabled pages and give an "unauthorized error"

Closes # .

### How to test the changes in this Pull Request:

1. Disable stock management
2. Go to dashboard home page
3. Check that no product cell is shown in Status widget

### Other information:




### FOR PR REVIEWER ONLY:
